packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -936,14 +1000,18 @@ describe('ReactDOMInput', () => {
 
   it('should warn if value is null', () => {
     expect(() =>
-      ReactTestUtils.renderIntoDocument(<input type="text" value={null} />),
+      ReactTestUtils.renderIntoDocument(
+        <input type="text" value={null} onChange={emptyFunction} />,
```

Comment:
Looks like these changes are now unnecessary and can be reverted? We still expect one warning with `null` even if you don't pass `onChange`.

Reply:
Ah yes gotcha 👍

Reply:
Ping :-)

Reply:
Good now!